content/en/platform/corda/4.11/community/api-vault-query.md

```diff
@@ -541,18 +541,25 @@ demonstrated in the following example.
 #### Query for all states using a pagination specification and iterate using the *totalStatesAvailable* field until no further pages available:
```

Comment:
Should we add a comment to say this will also detect if the vault has concurrently changed whilst querying and will try to get the latest up to date version?

Reply:
Will do.